alphadia/constants/keys.py

```diff
@@ -0,0 +1,24 @@
+class ConstantsClass(type):
+    """A metaclass for classes that should only contain string constants."""
+
+    def __setattr__(self, name, value):
+        raise TypeError("Constants class cannot be modified")
+
+    def get_values(cls):
+        """Get all user-defined string values of the class."""
+        return [
+            value
+            for key, value in cls.__dict__.items()
+            if not key.startswith("__") and isinstance(value, str)
+        ]
+
+
+class OutputKeys(metaclass=ConstantsClass):
```

Comment:
Should we rename this so it reflects it's used in the stats table?
like `StatsOutputKeys`